garak/generators/nim.py

```diff
@@ -77,7 +80,23 @@ def _call_model(
         if self.vary_temp_each_call:
             self.temperature = random.random()
 
-        return super()._call_model(prompt, generations_this_call)
+        try:
+            result = super()._call_model(prompt, generations_this_call)
+        #        except openai.NotFoundError as oe:
+        except Exception as oe:
+            msg = "NIM endpoint not found. Is it spelled correctly?"
+            logging.critical(msg, exc_info=oe)
+            print(f"🛑 {msg}")
+            raise GarakException(oe.__str__()) from oe
```

Comment:
Generators should not print. The string passed to `GarakException` will be printed by the CLI on exit.

```suggestion
            raise GarakException(msg) from oe
```